app/src/main/java/com/sunasterisk/fooddaily/ui/base/BaseFragment.kt

```diff
@@ -19,5 +28,32 @@ abstract class BaseFragment : Fragment() {
         return inflater.inflate(layoutRes, container, false)
     }
 
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        onFragmentInteractionListener = context as OnFragmentInteractionListener
```

Comment:
phải check xem context của nó đã implement interface chưa thì mới được cast
```suggestion
        if(context is OnFragmentInteractionListener) {
            interactionListener = context
        }
        
```